test/factories/cluster_group.rb

```diff
@@ -1,7 +1,7 @@
 FactoryBot.define do
   # creates a ClusterGroup object
   factory :cluster_group do
-    cluster_type { '2d' }
+    cluster_type { }
```

Comment:
remember, that's just saying the default is '2d', so I'd leave this as-is, as I think that's a sensible default